components/card/card.css

```diff
@@ -0,0 +1,138 @@
+.card{
+    display: flex;
+    border: 1px solid var(--primary-color);
+}
+.card-vertical{
+    flex-direction: column;
+    position: relative;
+    max-width: 302px;
```

Comment:
Avoid using absolute units, until it's much required.

Reply:
changed